packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -285,7 +285,7 @@ const styles = StyleSheet.create({
 });
 
 <View style={styles.row}>
-    <MultiSelect menuText="Empty" placeholder="empty" />
```

Comment:
Huh - was menuText a mistake?

Reply:
Apparently it was at least for Single and Multi Select, this value is extracted/configured internally and not exposed to the public API.